bedrock/firefox/views.py

```diff
@@ -655,3 +666,11 @@ def get_template_names(self):
             template = 'firefox/feedback/unhappy.html'
 
         return [template]
+
+
+def android(request):
+    # check for variant in querystring for send-to-device testing
+    variant = request.GET.get('v', '')
```

Comment:
Yes. Was about to say the same. We need to sanitize this input to only accept the values we expect.